Automattic/jetpack
# Jetpack Manage: Update the existing Manage banner in AAG section to the UpsellBanner component

Fixes https://github.com/Automattic/jetpack-manage/issues/238

## Proposed changes:

This PR updates the Manage banner in the Dashboard (Jetpack plugin) to use the `UpsellBanner` component instead of the current one, `Agencies_Card`. The banner's title copy remains the same:

<img width="1055" alt="image" src="https://github.com/Automattic/jetpack/assets/9832440/810e8c65-e1d2-41f3-b5a6-95db1b063ff4">

### Other information:


## Jetpack product discussion


Issue: https://github.com/Automattic/jetpack-manage/issues/238

## Does this pull request change what data or activity we track or use?



## Testing instructions:
- Check the code
- Apply this branch, check it, or launch a JN site.
- Connect the site with a WPCOM account (you will need a user with only 1 site connected and other with 2 or more sites connected to WPCOM.
   - A user with only 1 site: you will not see the banner.
   - A user with 2 or more sites: you will see the banner.
- If your user is an Agency or Internal, you will see different primary CTAs:
   - Change your partner account type to Internal <=> Agency: in the MC tool: /jetpack-partner-portal/partners 
      - As an agency, the CTA is `Manage sites`, and it will send you to the Manage dashboard.
      - If you are not already an Agency (Internal A8c), the CTA will be `Sign up for free`.
- **Note** If you test everything with the same WP site, you will have to purge the cache (transient) or set to false `$partner` this line `$partner = get_transient( 'jetpack_partner_data' );` in the `packages/my-jetpack/src/class-jetpack-manage.php`
